@navikt/ds-react/src/form/search-field/stories/search-field-example.tsx

```diff
@@ -7,24 +7,21 @@ import {
   SearchFieldClearButton,
 } from "../index";
 
-export const Example = () => {
+export const Example = ({ size = "medium" }: { size: "medium" | "small" }) => {
   const [value, setValue] = useState("");
 
   return (
-    <SearchField
-      label="Mollit eiusmod"
-      description="Ea cupidatat eu sunt commodo"
-    >
+    <SearchField size={size} label="Skriv i søkefeltet for å vise clearbutton">
       <SearchFieldInput
         value={value}
         onChange={(e) => setValue(e.target.value)}
       />
-      <SearchFieldClearButton onClick={() => setValue("")}>
+      <SearchFieldClearButton onClick={() => setValue("")} visible={!!value}>
```

Comment:
Hva om vi heller anbefaler å implementere `visible`-funksjonaliteten som dette: `{!!value && <SearchFieldClearButton ...`? Jeg tenker det gjør vår kode enklere, og vi slipper å ha en prop for å gjøre en komponent `display: none`.

Reply:
Jeg tenker det gjør APIet tydeligere også, for navnet `visible` kan misforstås som at knappen fortsatt er tilgjengelig for skjemlesere (noe den ikke er med `display: none`)

Reply:
👍 Enig, fikser den for det da.